package/templatectl/usr-share-remarkable-templates.mount

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Bind mount /opt/share/remarkable/templates over /usr/share/remarkable/templates
+DefaultDependencies=no
+Conflicts=umount.target
+Before=local-fs.target umount.target
```

Comment:
Just a nit: Why disable default dependencies and re-add them manually afterwards? If you just want to disable the `After=local-fs-pre.target` and `Before=local-fs.target` default dependencies, you can simple set the `nofail` mount option below.

Reply:
I stole this from the *.mount's in bootstrap/reenable.

Reply:
Okay, we’ll fix that at a later time then.